lib/date.rb

```diff
@@ -4,6 +4,7 @@
 require 'date_core'
 
 class Date
+  VERSION = '3.2.1' # :nodoc:
```

Comment:
In the normal pattern it would be defined in `lib/date/version.rb` and required in the gemspec, but I'm not sure what the consequence could be for something like `Date`. So I do some parsing instead.